docs/src/app/components/pages/components/SelectField/ExampleCustomLabel.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import SelectField from 'material-ui/lib/SelectField';
+import MenuItem from 'material-ui/lib/menus/menu-item';
+
+export default class SelectFieldCustomLabelExample extends React.Component {
```

Comment:
@oliviertassinari I think we should leave the discussion of this for when we want to move the docs to src.